packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -479,4 +480,59 @@ describe('masternode', () => {
       })
     })
   })
+
+  describe('utxosToAccount', () => {
+    let address1: string
+    let address2: string
+
+    beforeAll(async () => {
+      address1 = await container.call('getnewaddress')
+      address2 = await container.call('getnewaddress')
+    })
+
+    it('should utxosToAccount', async () => {
+      const balanceBefore = await container.call('getbalance')
+
+      const info: UtxosToAccountInfo = { details: [] }
+      info.details.push({ address: address1, amount: 5 })
+      info.details.push({ address: address2, amount: 5 })
+
+      const data = await client.account.utxosToAccount(info)
+
+      expect(typeof data).toBe('string')
+      expect(data.length).toBe(64)
+
+      const balanceAfter = await container.call('getbalance')
+      expect(balanceAfter).toBeLessThan(balanceBefore - 10)
```

Comment:
This line of code not always return true in my local environment. I am unsure why. Kindly advise

Reply:
I think I know why. It is impossible to prevent the master node from minting DFI.